docs/modules/ROOT/pages/spring-cloud-netflix.adoc

```diff
@@ -269,9 +269,9 @@ It is initialized in a `SmartLifecycle` (with `phase=0`), so the earliest you ca
 
 ==== Underlying HTTP clients
 
-`EurekaClient` uses either `RestTemplate`, `WebClient` or `JerseyClient` under the hood. In order to use the `EurekaClient`, you need to have one of the supported HTTP clients on your classpath.
+`EurekaClient` uses either `RestClient`, `RestTemplate`, `WebClient` or `JerseyClient` under the hood. In order to use the `EurekaClient`, you need to have one of the supported HTTP clients on your classpath.
 
-To use `RestTemplate`, add `spring-boot-starter-web` to your dependencies. To use `WebClient`, add `spring-boot-starter-webflux` to your dependencies. If both `RestTemplate` and `WebClient` are on the classpath when `eureka.client.webclient.enabled` is set to `true`, `WebClient` is used. Otherwise, `RestTemplate` is used.
+To use `RestTemplate` or `RestClient`, add `spring-boot-starter-web` to your dependencies. To use `WebClient`, add `spring-boot-starter-webflux` to your dependencies. If both `spring-boot-starter-web` and `spring-boot-starter-webflux` are included in the dependencies and the `eureka.client.webclient.enabled` is set to `true`, the `WebClient` will be used. On the other hand, if `eureka.client.restclient.enabled` is set to `true`, the `RestClient` will be used. Otherwise, the `RestTemplate` will be used.
```

Comment:
Please rephrase:
"If both `spring-boot-starter-web` and `spring-boot-starter-webflux` are included in the dependencies and the `eureka.client.webclient.enabled` is set to `true`, the `WebClient` will be used. On the other hand, if `eureka.client.restclient.enabled` is set to `true`, the `RestClient` will be used. Otherwise, the `RestTemplate` will be used."

to:

"If both `spring-boot-starter-web` and `spring-boot-starter-webflux` are included in the dependencies and the `eureka.client.webclient.enabled` flag is set to `true`, then `WebClient` will be used. If that's not the case and `eureka.client.restclient.enabled` is set to `true`, the `RestClient` will be used. Otherwise, the `RestTemplate` will be used."

Reply:
Fixed 😄